c/CMakeLists.txt

```diff
@@ -487,16 +492,18 @@ if (BUILD_STATIC_LIBS)
   target_compile_definitions(qpid-proton-static PUBLIC PROTON_DECLARE_STATIC)
   target_link_libraries (qpid-proton-static
     qpid-proton-core-static
-    $<$<BOOL:${HAS_PROACTOR}>:qpid-proton-proactor-static>
+    ${TARTET_qpid-proton-proactor-static}
     ${SSL_LIB} ${SASL_LIB} ${TIME_LIB} ${PLATFORM_LIBS} ${PROACTOR_LIBS})
+  set_target_properties(qpid-proton-static PROPERTIES
+    POSITION_INDEPENDENT_CODE ON)
 endif(BUILD_STATIC_LIBS)
 
 # Install executables and libraries
 if (BUILD_STATIC_LIBS)
   set(STATIC_LIBS qpid-proton-static qpid-proton-core-static)
 endif()
 install(TARGETS qpid-proton qpid-proton-core ${STATIC_LIBS}
-  EXPORT  proton
+  EXPORT  ProtonTargets
```

Comment:
Just naming pattern that the CMake docs used in its example. I don't think this changes anything regarding compatibility.

Reply:
It causes CMake to generate ProtonTargets.cmake, which I then include from the ProtonConfig.cmake file. It does the heavy lifting of locating the library files relative to the ProtonConfig.cmake. I'll attach that file

[ProtonTargets.txt](https://github.com/apache/qpid-proton/files/6480182/ProtonTargets.txt)
[ProtonTargets-relwithdebinfo.txt](https://github.com/apache/qpid-proton/files/6480183/ProtonTargets-relwithdebinfo.txt)
[ProtonConfig.txt](https://github.com/apache/qpid-proton/files/6480184/ProtonConfig.txt)
[ProtonConfigVersion.txt](https://github.com/apache/qpid-proton/files/6480185/ProtonConfigVersion.txt)
[FindCyrusSASL.txt](https://github.com/apache/qpid-proton/files/6480186/FindCyrusSASL.txt)